IdnoPlugins/OAuth2/Pages/Connect.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace IdnoPlugins\OAuth2\Pages {
+
+    class Connect extends \Idno\Common\Page
+    {
+
+        function getContent()
+        {
+            $this->gatekeeper();
+
+            $fwd = $this->getInput('fwd'); // return page
+            $client_id = $this->getInput('client_id');
+            $scope = $this->getInput('scope');
+
+            $client = \IdnoPlugins\OAuth2\Application::getOne(['key' => $client_id]);
+            if ($client) {
+
+                $t = \Idno\Core\site()->template();
+                $t->body = $t->__(array('fwd' => $fwd, 'client_id' => $client_id, 'scope' => $scope, 'client' => $client))->draw('oauth2/forms/connect');
+                $t->title = 'Authorise connection...';
+                $t->drawPage();
+            }
+            else
+            throw new \Exception(\Idno\Core\Idno::site()->language()->_("Could not load client associated with %s", [$client_id]));
```

Comment:
Doesn't need to be two lines / could be indented for clarity

Reply:
This appears to have been stripped by phpcbf, I think the rules need to be updated